src/components/bcourses/roster/RosterPhoto.vue

```diff
@@ -9,6 +9,7 @@
     eager
     :lazy-src="photoPlaceholder"
     width="72"
+    transition="null"
```

Comment:
@pauline2k I think this should be `:transition="null"`, otherwise it will have the string value "null".

Reply:
@lyttam Oh right! I think the effect in the browser is the same but amended anyhow :)